moveit_core/collision_detection/include/moveit/collision_detection/test_collision_common_pr2.h

```diff
@@ -112,6 +118,8 @@ TYPED_TEST_P(CollisionDetectorTest, LinksInCollision)
   collision_detection::CollisionResult res1;
```

Comment:
Possibly you're missing this:

req.group_name = parameters_.move_group_name;

Reply:
(no idea how the tests were running without this)

Reply:
The `CollisionRequest` does not set the `group_name`, and in doing so indicates that it should check the whole body for collisions. I figured it would be the same here.